mmv1/templates/terraform/examples/cloudbuildv2_connection.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_cloudbuildv2_connection" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  location = "us-central1"
+  name = "<%= ctx[:vars]['connection_name'] %>"
+
+  github_config {
+    app_installation_id = 0
+
+    authorizer_credential {
+      oauth_token_secret_version = "projects/gcb-terraform-creds/secrets/github-pat/versions/1"
```

Comment:
you can override this value to show something different in the example (but still use the valid value for the test).

See the repo_trigger example.

Reply:
@rileykarson correct me if I'm wrong, but the only example docs generated are small snippets of the IAM resources. I dont see anything else being generated in the provider, so this example is not actually in the docs.

Reply:
Correct! See https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-6778-old..auto-pr-6778#diff-dc882a2ee63445ca0f8023e556cf26711bb4ecf327c8a380736e118164b03278R36